### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by YaniGalyov**

### CODE REVIEW

File: Greetings.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import "fmt"
+
+func main() {
+
+    names := []string{"Yani", "Vlad", "Krasi", "Ivan", "Kris", "Naso", "Dimi", "Geri", "Lili", "Zoya"}
+
+    for i, c := range names {
+        fmt.Println("", i)
+        fmt.Println("", c)
```
Comment (krasi-georgiev) [inline Greetings.go]: Why do you have an empty string `""` in front ?

Comment (YaniGalyov) [reply Greetings.go]: Fix it :)

Comment (krasi-georgiev) [reply Greetings.go]: I don't see the changes, need to push the new  changes(new commit) to the "Greetings" branch.

Comment (YaniGalyov) [reply Greetings.go]: Do you see it now?

**Comment by krasi-georgiev** [comment]
LGTM :+1:

**Pull request merged**